src/main/cpp/t01_char.cpp

```diff
@@ -30,5 +30,7 @@
 using namespace std;
 
 int t01_char() {
-
+          string a, b = "1234567890";
+          cin >> a;
+          if (b.find(a) < 10) {cout << "yes";} else {cout << "no";}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Inefficient usage of string::find() in condition; string::starts_with() could be faster.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4494122)